### PR TITLE
Nerfs lichdom holoparasite combo

### PR DESCRIPTION
**Pull request by Moltijoe**
# Why is this good for the game?
Undying holoparasite that deletes the corpse of a dead lich
Makes it impossible to find the phylactery
Removes the downside of holoparasite

rather than removing the interesting combo, makes it less meta

:cl:  
tweak: Holoparasites no longer dust liches upon death
tweak: Holoparasites lose 4 random stats upon revival by a lich
/:cl:

### CODE REVIEW

**Comment by Therandomhoboo** [comment]
Agree with the PR, think losing 4 random stats is a bit too much though.

**Comment by RG4ORDR** [comment]
> Agree with the PR, think losing 4 random stats is a bit too much though.

Holoparas giving a very strong antag a buddy that can shoot funny crystal bolts or save their lives is extremely strong.

**Comment by Therandomhoboo** [comment]
> > Agree with the PR, think losing 4 random stats is a bit too much though.
> 
> Holoparas giving a very strong antag a buddy that can shoot funny crystal bolts or save their lives is extremely strong.

iirc the holoparasite spell isn't always ranged and have different stats & powers cause you can flick through 'em. Unless there are two spells for them. Either way, one death could neuter the holoparasite completely depending on rng for the stats provided.

Don't get me wrong, I agree with the PR but I think debuffing 4 stats randomly is too much. I would suggest 2. If not enough then make it that each subsequent death after the first one increases the amount of stats lost by an additional 1? If it isn't too much of a hassle to code.

**Comment by MajManatee** [comment]
I think the wording was poor, but this seems great,

File: yogstation/code/modules/guardian/guardianstats.dm [hunk 1]
```diff
@@ -75,3 +75,33 @@
 		var/datum/guardian_ability/minor/minor_ability = A
 		stats_info += minor_ability.name
 	return stats_info.Join(", ")
+
+/datum/guardian_stats/proc/weaken(times = 1)//reduces a random stat x times
+	for(var/i = 0; i < times; i++)
+		var/list/reduction = list("damage", "defense", "speed", "potential", "range")
+		shuffle(reduction)
+		var/found = FALSE
+		for(var/option in reduction)
```
Comment (SomeguyManperson) [inline yogstation/code/modules/guardian/guardianstats.dm:84]: suggest some failsafe in case all the stats end up at 1 and this loops

Comment (Moltijoe) [reply yogstation/code/modules/guardian/guardianstats.dm:84]: It should stop when it goes through all options, shouldnt it?

Comment (SomeguyManperson) [reply yogstation/code/modules/guardian/guardianstats.dm:84]: i mifhr vw crazy

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**